sdcm/report_templates/results_latency_during_ops.html

```diff
@@ -61,6 +61,23 @@ <h2>{{ operation }}</h2>
             {% endfor %}
         </div>
 
+    <div>
+        {% if last_events %}
+            <h3>
+                <span>Last events by severity</span>
+            </h3>
+            {% for severity, events in last_events.items() %}
+                <h4>
+                {{ severity }} - [{{ events_summary.get(severity, 0) }}]
+                </h4>
+                {% for event in events %}
+                    <pre>{{event}}</pre>
+                    <hr>
+                {% endfor %}
+            {% endfor %}
+        {% endif %}
+    </div>
+
     <h3>Links:</h3>
```

Comment:
if this code repeated in several  templates, may be  it is better to move it to separate file and just include it here?

Reply:
this was not yet done (i actually started, but didn't finish, and didn't test other templates that "will use it"

Reply:
change was done, i'm checking for regression on other templates that use the `results_base_custom.html` too